model/os/registry.yaml

```diff
@@ -59,6 +77,15 @@ groups:
             - id: zos
               value: 'zos'
               brief: "IBM z/OS"
+              deprecated: "Use `os.name` instead"
```

Comment:
`zos` is a real unique operating system type, I don't think we should be removing it.